system/indy-node-tests/TestLedgerSuite.py

```diff
@@ -244,12 +244,54 @@ class TestLedgerSuite:
                 {'support_revocation': revocation}
             )
         )
+        assert res3['op'] == 'REPLY'
 
         res4 = await ensure_get_something(get_cred_def, pool_handler, wallet_handler, trustee_did, cred_def_id_local)
 
         cred_def_id_ledger, cred_def_json = await ledger.parse_get_cred_def_response(json.dumps(res4))
         assert cred_def_id_local == cred_def_id_ledger
 
+    @pytest.mark.parametrize(
+        'target_role, tag, result',
+        [
+            ('ENDORSER', random_string(257), 'REQNACK'),
+            ('NETWORK_MONITOR', random_string(256), 'REJECT'),
+            (None, random_string(256), 'REJECT')
```

Comment:
Is 257 deliberate or just a typo?

Reply:
257 is boundary incorrect value to get REQNACK.